cards.js

```diff
@@ -25,18 +25,19 @@ class Card {
   }
   toString() {
     if(this.isNormal()) {
+      let symbol=this.getSuitSymbol();
       if(this.rank < 11) {
-        return this.rank + this.getSuitSymbol();
+        return this.rank + symbol;
       } else {
         switch(this.rank) {
           case 11:
-            return "J" + this.getSuitSymbol();
+            return "J" + symbol;
           case 12:
-            return "Q" + this.getSuitSymbol();
+            return "Q" + symbol;
           case 13:
-            return "K" + this.getSuitSymbol();
+            return "K" + symbol;
           case 14:
```

Comment:
Sim vajadzeetu buut 1, redzu ka tu neapmaniiji kluudu kuru es ielaidu 1. PR un peec tam salaboju savaa tagadeejaa PR.

Reply:
vai nu ar pamaniiji, tbqh es jau tagad vairs neatceros ko es domaaju kad so rakstiiju

Reply:
Vai tu to salaboji?

Reply:
Jap(#9